.pre-commit-config.yaml

```diff
@@ -370,8 +370,8 @@ repos:
         additional_dependencies: ['rich>=12.4.4']
       - id: update-setup-cfg-file
         name: Update setup.cfg file with all licenses
-        entry: ./scripts/ci/pre_commit/pre_commit_setup_cfg_file.sh
-        language: system
+        entry: ./scripts/ci/pre_commit/pre_commit_setup_cfg_file.py
```

Comment:
Should we remove `./scripts/ci/pre_commit/pre_commit_setup_cfg_file.sh` file?

Reply:
I thought I did :)

Reply:
Removed